tests/js/spec/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -190,4 +245,1168 @@ describe('WidgetBuilder', function () {
+  //     name: 'field',
+  //     at: 0,
+  //     control: true,
+  //   });
+
+  //   // user.display not should be filtered out for count_unique()
+  //   selectByLabel(wrapper, 'user.display', {
+  //     name: 'parameter',
+  //     at: 0,
+  //     control: true,
+  //   });
+
+  //   // Be able to choose a numeric-like option
+  //   selectByLabel(wrapper, 'measurements.lcp', {
+  //     name: 'parameter',
+  //     at: 0,
+  //     control: true,
+  //   });
+
+  //   await clickSubmit(wrapper);
+
+  //   expect(widget.displayType).toEqual('world_map');
+  //   expect(widget.queries).toHaveLength(1);
+  //   expect(widget.queries[0].fields).toEqual(['count_unique(measurements.lcp)']);
+  //   wrapper.unmount();
+  // });
+
+  // it('should filter y-axis choices by output type when switching from big number to line chart', async function () {
+  //   let widget = undefined;
+  //   const wrapper = mountModal({
+  //     initialData,
+  //     onAddWidget: data => (widget = data),
+  //   });
+  //   // No delete button as there is only one field.
+  //   expect(wrapper.find('IconDelete')).toHaveLength(0);
+
+  //   // Select Big Number display
+  //   selectByLabel(wrapper, 'Big Number', {name: 'displayType', at: 0, control: true});
+  //   expect(getDisplayType(wrapper).props().value).toEqual('big_number');
+
+  //   // Choose any()
+  //   selectByLabel(wrapper, 'any(\u2026)', {
+  //     name: 'field',
+  //     at: 0,
+  //     control: true,
+  //   });
+
+  //   selectByLabel(wrapper, 'id', {
+  //     name: 'parameter',
+  //     at: 0,
+  //     control: true,
+  //   });
+
+  //   // Select Line chart display
+  //   selectByLabel(wrapper, 'Line Chart', {name: 'displayType', at: 0, control: true});
+  //   expect(getDisplayType(wrapper).props().value).toEqual('line');
+
+  //   // Expect event.type field to be converted to count()
+  //   const fieldColumn = wrapper.find('input[name="field"]');
+  //   expect(fieldColumn.length).toEqual(1);
+  //   expect(fieldColumn.props().value).toMatchObject({
+  //     kind: 'function',
+  //     meta: {
+  //       name: 'count',
+  //       parameters: [],
+  //     },
+  //   });
+
+  //   await clickSubmit(wrapper);
+
+  //   expect(widget.displayType).toEqual('line');
+  //   expect(widget.queries).toHaveLength(1);
+  //   expect(widget.queries[0].fields).toEqual(['count()']);
+  //   wrapper.unmount();
+  // });
+
+  it('should automatically add columns for top n widget charts', async function () {
+    const defaultWidgetQuery = {
+      name: '',
+      fields: ['title', 'count()', 'count_unique(user)', 'epm()', 'count()'],
+      conditions: 'tag:value',
+      orderby: '',
+    };
+
+    renderTestComponent({
+      query: {
+        source: DashboardWidgetSource.DISCOVERV2,
+        defaultWidgetQuery: urlEncode(defaultWidgetQuery),
+        displayType: DisplayType.TOP_N,
+        defaultTableColumns: ['title', 'count()', 'count_unique(user)', 'epm()'],
```

Comment:
Just to confirm, this test is asserting that it'll pull values from the query params right? I think we should call that out in the title